third_party/terraform/resources/resource_dataproc_cluster.go

```diff
@@ -144,7 +144,7 @@ func resourceDataprocCluster() *schema.Resource {
 									},
 
 									"tags": {
-										Type:     schema.TypeList,
+										Type:     schema.TypeSet,
```

Comment:
A `TypeSet` in a `TypeList` 🤔. I'd like a test to be using this field, even if a simple type doesn't take nearly as much work as a complex type.

Reply:
Done.